src/HealthChecks/HealthChecks/src/DefaultHealthCheckService.cs

```diff
@@ -218,6 +218,7 @@ private static partial class Log
         private const string HealthCheckEndText = "Health check {HealthCheckName} with status {HealthStatus} completed after {ElapsedMilliseconds}ms with message '{HealthCheckDescription}'";
 
 #pragma warning disable SYSLIB1006
+#pragma warning disable SYSLIB1025
```

Comment:
I'm not sure about reusing the same error ID with different log levels. I don't think we do this anywhere else. Suppressing now is fine, but should look into improving this in the future: https://github.com/dotnet/aspnetcore/issues/46099